internal/gatewayapi/helpers.go

```diff
@@ -109,7 +109,7 @@ func KindDerefOr(kind *gwapiv1.Kind, defaultKind string) string {
 // to a Gateway with the given namespace/name, irrespective of whether a
 // section/listener name has been specified (i.e. a parent ref to a listener
 // on the specified gateway will return "true").
-func IsRefToGateway(parentRef gwapiv1.ParentReference, gateway types.NamespacedName) bool {
+func IsRefToGateway(parentRefNamespace gwapiv1.Namespace, parentRef gwapiv1.ParentReference, gateway types.NamespacedName) bool {
```

Comment:
should we rename `parentRefNamespace` to `routeNamespace` in this diff, so its clear the value is derived from the namespace of the route, not the parentRef itself (like a Gateway or Listener) ?

Reply:
updated